web/components/AccessManagement/AccountTable.tsx

```diff
@@ -36,36 +36,40 @@ function AccountTable(props: PropertyType) {
         updateItemsToDelete={updateItemsToDelete}
       ></TableHeader>
       {!isLoading ? (
```

Comment:
So I tried that but sometimes while the spinner is still loading, it displays the account list or the no results found along with the spinner- and I wanted to avoid that.

Reply:
Let me know if you still want to change it or keep it as it is.

Reply:
I think we'll refactor this page so you can leave it like that - thanks!